Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -220,8 +219,8 @@ struct PerWebProcessState {
     std::optional<CGRect> frozenVisibleContentRect;
     std::optional<CGRect> frozenUnobscuredContentRect;
 
-    WebKit::TransactionID firstPaintAfterCommitLoadTransactionID;
-    WebKit::TransactionID lastTransactionID;
+    std::optional<WebKit::TransactionID> resetViewStateAfterTransactionID;
```

Comment:
Could we use `Markable<>` instead?

Reply:
Unfortunately `MonotonicObjectIdentifier` doesn't support `Markable` (because 0 is a valid value).

It's possible we that could make int max or something (the hash table deleted value) also the markable empty value? Feels like a bit of a hack, but also unlikely to to collide with 64 bits.